node_modules/oae-core/footer/footer.html

```diff
@@ -4,12 +4,30 @@
 <footer id="footer-container" class="text-center"><!-- --></footer>
 
 <div id="footer-template"><!--
-    <a href="http://apereo.org/" target="_blank" title="__MSG__COPYRIGHT__" class="oae-plain-link">
-        __MSG__COPYRIGHT_SYMBOL__
-        __MSG__APEREO_FOUNDATION__
-        ${currentYear}.
-        __MSG__ALL_RIGHTS_RESERVED__
-    </a>
+    {var instanceName = oae.api.config.getValue('oae-tenants', 'instance', 'instanceName')}
+    {var instanceURL = oae.api.config.getValue('oae-tenants', 'instance', 'instanceURL')}
+    {var instanceLink = instanceName}
+    {if instanceURL}
+        {var instanceLink = '<a href="' + instanceURL + '" class="oae-plain-link">' + instanceName + '</a>'}
+    {/if}
+
+    {var hostingOrganisation = oae.api.config.getValue('oae-tenants', 'instance', 'hostingOrganisation')}
+    {var hostingOrganisationURL = oae.api.config.getValue('oae-tenants', 'instance', 'hostingOrganisationURL')}
+    {var hostingOrganisationLink = hostingOrganisation}
+    {if hostingOrganisationURL}
+        {var hostingOrganisationLink = '<a href="' + hostingOrganisationURL + '" class="oae-plain-link">' + hostingOrganisation + '</a>'}
+    {/if}
```

Comment:
I'd prefer if all this logic could go in a JS file instead of the template. It will be much more readable and documentable. You could build 2 objects in JS:

`instance` and `hostingOrganisation` and pass it in as data to the template.